lib/Compiler.js

```diff
@@ -134,6 +134,10 @@ class Compiler extends Tapable {
 		this.requestShortener = new RequestShortener(context);
 
 		this.running = false;
+
+		// Added to make TypeScript happy
```

Comment:
This type of comment is not needed. Property should always be defined in order to avoid v8 transitions